src/lab13.py

```diff
@@ -1045,8 +776,6 @@ def render(self):
             self.needs_paint = False
 
         clamped_scroll = self.clamp_scroll(self.scroll)
-        if clamped_scroll != self.scroll:
```

Comment:
Why is this no longer needed?

Reply:
If you look a line or two down, this `if` statement is duplicated. It doesn't have any effects we need to repeat twice, either. Probably a copy-and-paste bug.